portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetAdmin.testcase

```diff
@@ -0,0 +1,77 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property osgi.modules.includes = "frontend-data-set-sample-web";
+	property portal.acceptance = "true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Frontend Dataset";
+	property testray.main.component.name = "User Interface";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-174054: Display a list of headless resources in new FDS View modal"
+	@priority = "5"
+	test CanDisplayHeadlessDeliveryAPIResources {
+		property custom.properties = "feature.flag.LPS-164563=true";
+
+		task ("Given access Datasets admin page") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Object",
+				panel = "Control Panel",
+				portlet = "Datasets");
+		}
+
+		task ("When go to add Datasets") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("Then displays list of headless resources") {
+			Click(locator1 = "CreateObject#OBJECT_LABEL");
+
+			AssertElementPresent(
+				key_item = "Liferay Headless Delivery",
+				locator1 = "ManagementBar#ACTIVE_DROPDOWN_ITEM");
+		}
+	}
+
+	@description = "LPS-174054: Display a list of headless resources in new FDS View modal"
+	@priority = "3"
+	test CanReturnNoMatchedResults {
+		property custom.properties = "feature.flag.LPS-164563=true";
+
+		task ("Given access Datasets admin page") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Object",
+				panel = "Control Panel",
+				portlet = "Datasets");
+		}
+
+		task ("When go to add Datasets") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("And When search for a phrase that doesn't match available headless providers") {
+			FrontendDataSetAdmin.searchProvider(key_provider = "apple");
+		}
+
+		task ("Then suggested list contains 0 results") {
+			AssertElementNotPresent(
+				key_tagName = "${key_provider}",
+				locator1 = "AssetCategorization#TAG_AUTOCOMPLETE_SPECIFIC");
```

Comment:
```suggestion
				key_provider = "apple",
				locator1 = "AssetCategorization#TAG_AUTOCOMPLETE_SPECIFIC");
```

Reply:
the variable `key_provider` is not global

Reply:
It doesn't work locally... you can check the log [here](https://gist.github.com/eltonccdantas/70bca54adb967fed6b18e058bed09876)

Reply:
I think it's something related to the Poshi Update, let's try it tomorrow